doc/developer-guide/api/functions/TSVConnSslFdGet.en.rst

```diff
@@ -0,0 +1,34 @@
+.. Licensed to the Apache Software Foundation (ASF) under one or more
+   contributor license agreements.  See the NOTICE file distributed
+   with this work for additional information regarding copyright
+   ownership.  The ASF licenses this file to you under the Apache
+   License, Version 2.0 (the "License"); you may not use this file
+   except in compliance with the License.  You may obtain a copy of
+   the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+   implied.  See the License for the specific language governing
+   permissions and limitations under the License.
+
+
+TSVConnSslFdGet
+===============
+
+Synopsis
+--------
+
+.. code-block:: cpp
+
+    #include <ts/ts.h>
+
+.. c:function:: int TSVConnSslFdGet(TSVConn sslp)
+
+
+Description
+-----------
+Returns the file descriptor associated with the SSL connection :arg:`sslp`.
```

Comment:
Why do we need to limit this just for SSL connection? Are we going to add `TSVConnFdGet` and `TSVConnQuicFdGet` if we needed? Plugins would need to call isSSL() and isQUIC() then call the corresponding TSVConnXFdGet()`. It doesn't sounds handy.

Reply:
Yeah, that crossed my mind as I pushed the branch up.  Let me take a pass at generalizing.